.gitignore

```diff
@@ -2,35 +2,31 @@
 __pycache__/
 *.py[cod]
 
-# Idea software family
-.idea/
```

Comment:
why was it necessary to reorder the `.gitignore` file?

Reply:
It wasn’t strictly necessary - I just had to reorder what I needed. I can revert!